contrib/google.golang.org/grpc/grpc.go

```diff
@@ -38,5 +38,5 @@ func finishWithError(span ddtrace.Span, err error) {
 		err = nil
 	}
 	span.SetTag(tagCode, errcode.String())
-	span.Finish(tracer.WithError(err))
+	span.Finish(tracer.WithError(err), tracer.NoBacktrace())
```

Comment:
This is fine but it can't be enabled by default. We'll need an interceptor option `NoDebugStack`.